service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -78,6 +89,19 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
             .build();
     }
 
+    private String getFirstHearingStartDate(List<Element<HearingBooking>> hearings) {
+        String hearingDate;
+
+        try {
+            LocalDateTime startDate = hearingBookingService.getMostUrgentHearingBooking(hearings).getStartDate();
+            hearingDate = formatLocalDateTimeBaseUsingFormat(startDate, FULL_DATE);
+        } catch (IllegalStateException e) {
+            hearingDate = "Please enter a hearing date";
+        }
```

Comment:
ideally service should return Optional and client should decide how to handle this